theorems/T000772.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: T000772
+if:
+  and:
+  - P000053: true
+  - P000146: true
+then:
+  P000220: true
+---
+
+Let $d$ be a metric, and define $\mathcal{U}_0 = \{X\}$ and $\mathcal{U}_{n+1}$ to be open partition which is a refinement of $\{B_d(x, \frac{1}{n})\cap U : x\in X, U\in\mathcal{U}_n\}$. Define $\delta(x, y) = \inf\{1/n : \exists_U\ \{x, y\}\subseteq U\in \mathcal{U}_n\}$.
```

Comment:
```suggestion
Let $d$ be a metric on $X$. Define $\mathcal{U}_1 = \{X\}$ and $\mathcal{U}_{n+1}$ to be an open partition which is a refinement of $\{B_d(x, \frac{1}{n})\cap U : x\in X, U\in\mathcal{U}_n\}$. Define $\delta(x, y) = \inf\{1/n : \{x, y\}\subseteq U \text{ for some }U\in\mathscr U_n \}$.
```
need to start at $n=1$ since we use $1/n$.